### PR TITLE
Smoke Tests: update requirements versions to DO NOT SQUASH

### PR DESCRIPTION
**Pull request by danieljurek**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by danieljurek** [comment]
DO NOT SQUASH this PR. Merge both commits in if possible. This ensures we have a history of working smoke tests for all released previews.

File: samples/smoketest/event_hubs_async.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def __init__(self):
         connection_string = os.environ["EVENT_HUBS_CONNECTION_STRING"]
         event_hub_name = "myeventhub"
         self.client = EventHubClient.from_connection_string(
-            connection_string, event_hub_name
+            connection_string, event_hub_path=event_hub_name
```
Comment (scbedd) [inline samples/smoketest/event_hubs_async.py]: This is for an API Change in eventhub right? (the change from positional to keyword arg)

Comment (danieljurek) [reply samples/smoketest/event_hubs_async.py]: It is :)

**Pull request merged**